changelog/@unreleased/pr-1135.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: <!-- User-facing outcomes this PR delivers -->
```

Comment:
huh, I think the robot has been automatically committing these lately

Reply:
Nah, I think I just didn't fill this stuff out 🤦